### PR TITLE
New ChirpRadio connector

### PR DESCRIPTION
**Pull request by jbwharris**
New connector for https://chirpradio.org

image

### CODE REVIEW

File: src/core/connectors.ts [hunk 1]
```diff
@@ -9,6 +9,11 @@ export interface ConnectorMeta {
 	 * true if connector uses blocklist. Connector must implement {@link Connector.getChannelId}
 	 */
 	usesBlocklist?: true;
+
+	/**
+	 * true if website has its own scrobbling system the user needs to be aware of.
+	 */
+	hasNativeScrobbler?: true;
```
Comment (inverse) [inline src/core/connectors.ts:16]: This file looks like it needs to be adapted to only the addition of the connector addition

**Review by inverse — CHANGES_REQUESTED** [review]
Looks good aside the additional changes in `src/core/connectors.ts`

**Comment by jbwharris** [comment]
@inverse I just pushed over the change to connectors.ts

**Comment by inverse** [comment]
@jbwharris looks like there are still a lot of changes that shouldnt be there :thinking:

**Comment by jbwharris** [comment]
@inverse I have no idea what I've done wrong with this one. I'm going to create a new branch and do a new PR

**Pull request closed without merging**